docs/Converters/UKBtoBIDS.md

```diff
@@ -0,0 +1,87 @@
+<!-- markdownlint-disable MD046 -->
+# `ukb-to-bids` – Conversion of the UK Biobank (UKB) to BIDS
+
+!!! quote "Description reproduced from the [UK Biobank webpage](https://www.ukbiobank.ac.uk/)"
+    UK Biobank is a large-scale biomedical database and research resource, containing in-depth genetic and health information from half a million UK participants. The database is regularly augmented with additional data and is globally accessible to approved researchers undertaking vital research into the most common and life-threatening diseases. It is a major contributor to the advancement of modern medicine and treatment and has enabled several scientific discoveries that improve human health
+
+## Dependencies
+
+If you installed the core of Clinica, this converter needs no further dependencies.
+
+## Downloading UK Biobank
+
+The UK Biobank to BIDS converter requires the user to have downloaded the UK Biobank imaging and clinical data.
+
+Once you have access to it, the imaging data can be downloaded and extracted using this tutorial: https://biobank.ctsu.ox.ac.uk/~bbdatan/Accessing_UKB_data_v2.3.pdf, and the clinical data can be extracted using the information from this repository https://github.com/kenhanscombe/ukbtools.
+
+
+Be careful about two things: 
+    - the converter only takes in the nifti provided by UK Biobank, not the dicoms.
+    - the converter will not work if you do not have among the clinical data the year of birth, the age at recruitment and at sessions and the sex of the subject.
+
+## Supported modalities
+
+Please note that this converter processes the following modalities : 
+- T1W
+- T2 Flair
+- DWI
+- SWI
+- tfMRI
+- rsfMRI
+
+!!! Chosen files
+    Whenever possible, we use the rawest files in order not to impose opinions onto the user. When available we get the associated json.
```

Comment:
"in order not to impose opinions" --> "so as to leave it to the user to choose the processing needed"